automated-testing/ab-testing/readme.md

```diff
@@ -0,0 +1,23 @@
+# ~Test type~
+
+The document should start with a brief overview about the test type and what is covered in this document, the goal here is to provide a high-level description to help the reader understand what is covered to decide whether to continue reading or not.
```

Comment:
This is a question to debate, but should we add documents as TODO:s? It is not completely apparent that this is a TODO until you start reading more thoroughly.  

IMHO at any point in time **master** is "shippable" so an alternative way to fix this is by instead creating issues for the items that need to be filled in, with templates in the issues, and either leave empty documents - perhaps saying "Information on this topic will be added soon" - or maybe not even adding links right now, but noting in the tasks that links, and docs need to be added.

Reply:
Hello @TessFerrandez ,

The idea of this PR was to create placeholder pages with the template we picked up so for the contributors that will finish each of the pages will be straightforward to follow the same structure.

Thank you!

Reply:
I saw that you started a conversation on teams about this, and I think it's fair if the other changes will come in immediately.  Just wanted to bring it up as this goes against our own best practices:

https://github.com/microsoft/code-with-engineering-playbook/blob/master/source-control/readme.md
- Commit complete and well tested code. Never commit incomplete code, get in the habit of testing your code before committing.